pg/pg.go

```diff
@@ -27,7 +29,59 @@ func NewPG() (*PG, error) {
 		return &PG{}, fmt.Errorf("PG ERROR: Could not open connection to postgres \n %s", err)
 	}
 
-	return &PG{
-		Conn: db,
-	}, nil
+	res := &PG{
+		Conn:     db,
+		QueryMap: make(map[string]string, 0),
```

Comment:
I feel a little weird about runtime-accessing queries via string key map. I feel like this necessitates a lot of extra key-presence checks